core/src/queue.rs

```diff
@@ -223,7 +223,7 @@ impl Queue {
     pub fn push(&self, tx: AcceptedTransaction, wsv: &WorldStateView) -> Result<(), Failure> {
         trace!(?tx, "Pushing to the queue");
         if let Err(err) = self.check_tx(&tx, wsv) {
-            warn!("Failed to evaluate signature check");
+            warn!("Failed to evaluate signature check. Error = {}", err);
```

Comment:
```suggestion
            warn!(Error=%err, "Failed to evaluate signature check);
```

I think it's more convinient to use `tracing` builtin things